bittensor/_tokenizer/__init__.py

```diff
@@ -14,42 +16,44 @@
 # THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION 
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
 # DEALINGS IN THE SOFTWARE.
-import bittensor
+
 from transformers import GPT2Tokenizer
 
 import bittensor 
 
 class tokenizer:
-
+    """ Implementation of the bittensor tokenizer
+    """
     cached_tokenizer_for_version: dict = {}
 
     def __new__( cls, version: str = None ):
         if version == None:
             version = bittensor.__version__
         if version not in cls.cached_tokenizer_for_version:
-            tokenizer = cls.get_tokenizer_for_version( version )
-            cls.cached_tokenizer_for_version[ version ] = tokenizer
+            _tokenizer = cls.get_tokenizer_for_version( version )
+            cls.cached_tokenizer_for_version[ version ] = _tokenizer
         else:
-            tokenizer = cls.cached_tokenizer_for_version[ version ]
-        return tokenizer
+            _tokenizer = cls.cached_tokenizer_for_version[ version ]
+        return _tokenizer
         
     # Tokenizer
     # NOTE (const): tokenizers are guaranteed to improve and expand as time progresses. We version the tokenizer here.
     # neurons must be aware that versions will increase and be ready to convert between tokenizers.
     # TODO (const): Add functionality to allow tokenizer conversion. i.e. for input token conversion.
     @classmethod
     def get_tokenizer_for_version( cls, version = bittensor.__version__ ):
-
-        tokenizer = GPT2Tokenizer.from_pretrained("gpt2", local_files_only=False)
-        tokenizer.padding_side = "left"
-        tokenizer.add_prefix_space = False
-        tokenizer.add_special_tokens({'bos_token': "[BOS]"}) # A special token representing the beginning of a sentence.
-        tokenizer.add_special_tokens({'eos_token': "[EOS]"}) # A special token representing the end of a sentence.
-        tokenizer.add_special_tokens({'unk_token': "[UNK]"}) # A special token representing an out-of-vocabulary token.
-        tokenizer.add_special_tokens({'sep_token': "[SEP]"}) # A special token separating two different sentences in the same input (used by BERT for instance)
-        tokenizer.add_special_tokens({'pad_token': "[PAD]"}) # A special token used to make arrays of tokens the same size for batching purpose. Will then be ignored by attention mechanisms or loss computation.
-        tokenizer.add_special_tokens({'cls_token': "[CLS]"}) # A special token representing the class of the input (used by BERT for instance).
-        tokenizer.add_special_tokens({'mask_token': "[MASK]"}) # A special token representing a masked token (used by masked-language modeling pretraining objectives, like BERT).
+        """ Return the GPT2 tokenizer with bittersor's special tokens
```

Comment:
small nit, technically, its special tokens from all other major models (Bert,XLM, etc).

Reply:
oh it is not removed, just that the name was changed to _tokenizer because "tokenizer" this name is exactly the same as its class name